### ISSUE TITLE
Unharvestable Nimroot and Snowberries and some crops

### ISSUE THREAD

**Issue by nihues** [TitansBane/Elysium#50]
Just to add on the Issues, it's reported on discord.

Noticed Nimroot and Snowberries can't be harvested anywhere.

Some plants like leek/cabbage don't show to harvest also, even after talking to farmer, I've got the Riften home and can't harvest from my land too.

**Comment by burnskyc** [comment]
I've got the same issue. Any fixes or ideas?

**Comment by nihues** [comment]
I've noticed that if you go sneak you can harvest normally.

**Comment by burnskyc** [comment]
Yeah, so I checked the Discord and that's what they said. It has something to do with a mod that makes you ask farmers for permission to harvest, so you have to sneak to harvest. This includes nirnroot for some weird reason.
